### PR TITLE
refactor: track tokens per profile

### LINKED ISSUE #226: ️🛠 - Store imported ERC20 tokens on whole profile
### Task description

To prevent edgecases, it is the easiest to store erc20 on a profile level instead of accounts.

### Requirements

- [x] remove imported erc20 tokens on account
- [x] store erc20 tokens on profile
- [x] hide erc20 tokens with balance 0

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/popups/ImportErc20TokenFormPopup.svelte [hunk 1]
```diff
@@ -29,12 +29,12 @@
...
                 const erc20TokenMetadata = await getErc20TokenMetadata(tokenAddress, chainId)
                 if (erc20TokenMetadata) {
-                    updateActiveAccountTrackedTokens(tokenAddress, chainId)
+                    updateActiveProfileTrackedTokens(tokenAddress, chainId)
                     showAppNotification({
                         type: 'success',
                         alert: true,
                         message: localize('popups.importErc20Token.success', {
-                            values: { tokenSymbol: erc20TokenMetadata.symbol },
+                            values: { tokenSymbol: 'erc20TokenMetadata.symbol' },
```
Comment (Tuditi) [inline packages/desktop/components/popups/ImportErc20TokenFormPopup.svelte]: Is this line intended?

**Review by Tuditi — CHANGES_REQUESTED** [review]
One change requested, but looks good overall. I couldn't test it, since I cannot get ERC-20 tokens atm.

File: packages/desktop/components/popups/ImportErc20TokenFormPopup.svelte (hunk 1 above)
Comment (MarkNerdi) [reply packages/desktop/components/popups/ImportErc20TokenFormPopup.svelte]: no :sweat:

**Review by Tuditi — DISMISSED** [review]
LGTM! :fire:

File: packages/desktop/views/dashboard/send-flow/views/TransactionSummaryView.svelte [hunk 2]
```diff
@@ -142,6 +142,9 @@
             }
             await _onMount()
         } catch (err) {
+            const a = 2
+            console.error(a)
+
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/send-flow/views/TransactionSummaryView.svelte]: I think this should be removed?

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**